docs/installing-northstar/using-northstar/commands.md

```diff
@@ -49,6 +49,7 @@ bind "F5" "toggleconsole"
 | `give mp_weapon_peacekraber`         | Gives you peacekraber                                                                                       |                                                 |
 | `r_latencyflex`                      | Enables [LatencyFleX](../../using-northstar/playing-on-linux/#latencyflex) (Linux-only, enabled by default) | `0` / `1`                                       |
 | `rui_drawEnable`                     | Enables or disables the HUD                                                                                 | `0` / `1`                                       |
+| `r_drawViewModel`                    | Enables or disables viewing of the player's hands and weapons                                                | `0` / `1`                                       |
```

Comment:
```suggestion
| `r_drawviewmodel`                    | Enables or disables viewing of the player's hands and weapons                                                | `0` / `1`                                       |
```
convars arent case sensitive when being typed into console, but this should still be accurate